src/components/views/rooms/RoomList.tsx

```diff
@@ -98,44 +97,10 @@ interface ITagAesthetics {
     defaultHidden: boolean;
 }
 
-interface ITagAestheticsMap {
+const TAG_AESTHETICS: {
```

Comment:
Removing the existing dial pad button involved removing a lot of the changes in https://github.com/matrix-org/matrix-react-sdk/pull/5523/files#diff-f58269671c69c23ef15a599c0ba04d5aa22016ab65b79832e84a76b9114777bcR215

However, I'm not entirely sure about the `TAG_AESTHETICS` bits and whether that was used to hide VoIP-related rooms, or simply used for the context menu. I've removed them, and there seems to be no adverse effects - but I'd like to double-check that those changes weren't necessary to hide PSTN rooms.

Reply:
can we keep the interface definition? It makes the TS slightly less daunting to look at.

TagAesthetics are responsible for mostly what the `+` button does on a sublist, if anything, and naming for the default lists. It should be kept.

Reply:
I've kept the interface definition, but removed all modifications of it - which I believe is correct?

Reply:
I think that's fine - I didn't see anything particularly concerning about the removal.